rules/cross-platform/impact_hosts_file_modified.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/07/07"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/07/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+The hosts file on endpoints is used to control manual IP address to hostname resolutions. The hosts file is the first
+point of lookup for DNS hostname resolution so if adversaries can modify the endpoint hosts file, they can route traffic
+to malicious infrastructure. This rule detects modifications to the hosts file on Microsoft Windows, Linux (Ubuntu or
+RHEL) and macOS systems.
+"""
+index = ["auditbeat-*", "winlogbeat-*"]
```

```suggestion
index = ["auditbeat-*", "winlogbeat-*", "logs-endpoint.events.*"]
```

This logic would work on the endpoint agent as well